imap_processing/codice/tests/test_decom.py

```diff
@@ -0,0 +1,96 @@
+""" This function reads in a binary file and returns the binary data as a byte array"""
```

Comment:
Again, I think it would be useful at some point to refactor this in more of a traditional unit test style. Technically, running `pytest` over this file right now will not actually invoke any code since nothing is named `test_<something>`.  I think that is totally fine for now, I just wanted to point it out as a future improvement.